python/cugraph/cugraph/community/louvain.py

```diff
@@ -21,7 +21,8 @@
 from pylibcugraph import ResourceHandle
 
 
-def louvain(G, max_iter=100, resolution=1.0):
+# FIXME: max_iter should be renamed to max_level
```

Comment:
I'm not sure how to do this cleanly in python.  We want to rename the parameter, but in a safe way (deprecating the old name and continuing to support it for a release.

Same FIXME is in place for the task version.

@rlratzel

Reply:
Updated as suggested